build-system/tasks/performance/config.json

```diff
@@ -1,5 +1,15 @@
 {
   "headless": true,
   "runs": 100,
-  "urls": []
+  "urls": [],
```

Comment:
Yes, that's what it does. Would it be more clear to have `noHandlerUrls: []` instead of making it an object, because right now I don't see any other config data that fits in to the `noHandlers` category (`headless` and `runs` applies to *all* urls)?

Reply:
I find the declaration a bit confusing because `headless` and `runs` applies to all urls, but the top level `urls` doesn't. The mix and match makes the configuration structure not intuitive imo. 

I don't have strong opinion as long as everything agrees with the config setting.

Reply:
Hmm what do you guys think of the following:
```
urls: {
  'http://normal.test',
  'http://analytics-1.test': {handler: analyticsHandler},
  'http://analytics-2.test': {handler: analyticsHandler},
  'http://ads-1.test': {handler: adsHandler},
}
```

I got the inspo from eslint configs.

Reply:
@estherkim I like it, but would it have `'http://normal.test': {},`?

Reply:
Yeah hmm. Could be ok?

Reply:
I would still prefer categorize by handler, because it is more often that one wants to test all handler related urls locally. But this would also work!

Reply:
@kevinkimball Any preference between 
```
urls: {
  'http://normal.test',
  'http://analytics-1.test': {handler: analyticsHandler},
  'http://analytics-2.test': {handler: analyticsHandler},
  'http://ads-1.test': {handler: adsHandler},
}
```
vs
```
"noHandlers": {
  "urls": [],
},
"analyticsHandler": {
  "urls": [],
},
```